velox/exec/Aggregate.cpp

```diff
@@ -53,25 +53,27 @@ getAggregateFunctionEntry(const std::string& name) {
       });
 }
 
-AggregateRegistrationResult registerAggregateFunction(
+AggregateRegistrationResult registerAggregateFunctionWithMetadata(
```

Comment:
Curious, why couldn't you keep the original function name?

Reply:
Done.